<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyEventOptions</class>
 <widget class="QDialog" name="KeyEventOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-image: url(:/new/prefix1/Assets/Background1.jpg) }


QLabel{color:green; background-color:rgb(226, 226, 226); }


QPushButton { color: green; background-color:white;  font: 14pt &quot;Arial Rounded MT Bold&quot;;}

QComboBox {color:green; background-color:white}</string>
  </property>
  <widget class="QPushButton" name="btnAnalyse">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Analyse</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbEvent">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblEvent">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial Rounded MT Bold&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>Event</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblComp">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial Rounded MT Bold&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>Competition</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbComp">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblWithin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial Rounded MT Bold&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>Within</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbWithin">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblTime">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial Rounded MT Bold&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbTime">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblHeader">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Arial Rounded MT Bold&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>Key Event Analysis</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCompare">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial Rounded MT Bold&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>Compare To</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbCompare">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
